csharp/src/Microsoft.ML.OnnxRuntime/SessionOptions.shared.cs

```diff
@@ -245,7 +281,7 @@ public void AppendExecutionProvider_Tensorrt(OrtTensorRTProviderOptions trtProvi
 #if __MOBILE__
             throw new NotSupportedException("The TensorRT Execution Provider is not supported in this build");
 #else
-            NativeApiStatus.VerifySuccess(NativeMethods.SessionOptionsAppendExecutionProvider_TensorRT(handle, trtProviderOptions.Handle));
+            NativeApiStatus.VerifySuccess(NativeMethods.SessionOptionsAppendExecutionProvider_TensorRT_V2(handle, trtProviderOptions.Handle));
```

Comment:
CC: @chilo-ms . I think it is semantically right only if this is `SessionOptionsAppendExecutionProvider_TensorRT_V2()` because it is that API that consumes the new struct that is constructed in C#

Reply:
You are right. It should be SessionOptionsAppendExecutionProvider_TensorRT_V2(). Thanks for pointing it out. I think the reason that it still works is because old struct and new struct are the same.